helm-chart/values.yaml

```diff
@@ -222,6 +222,57 @@ cvat:
           protocol: TCP
           name: http
 
+  kvrocks:
+    enabled: true
+    external:
+      host: 127.0.0.1
```

Comment:
This doesn't seem like a useful default... how could an external Kvrocks be located at localhost?